### PR TITLE
Bugfix/updated business arch: Updated business architecture diagram

### LINKED ISSUE #17: [Eco Pass KIT v2.0.0] Update business architecture diagram
# What needs to be done?

- [x] Add The DPP aspect to the diagram
- [x] Add referers to the "EDC discovery"

### PR DESCRIPTION
**Pull request by matbmoser**
# Why we create this PR?
 
The business architecture is not reflecting the updated reality of the digital product pass parallel with the EcoPass
 
# What we want to achieve with this PR?
 
We want to update the diagram at the adoption view
 
# What is new?
 
## Updated
- Updated business context diagram at the adoption view

## Issues Fixed
- Fixed links in the images

# Related Issues

Closes #17

### CODE REVIEW

**Comment by matbmoser** [comment]
I think we should rename the digital twin registy to Digital Twin KIT for respecting the semantic of the naming

File: docs/page-software-development-view.md
Comment (maxim-bo) [inline docs/page-software-development-view.md:1]: I do see the text in the source diff view, however, if I switch to the rich diff view here, I just see both pictures without the text block in between. I assume its just a visualization thing and that the final file should be displayed correctly

File: docs/resources/adoption-view/adoption-view-EcoPassKITBusinessArch.svg
Comment (maxim-bo) [inline docs/resources/adoption-view/adoption-view-EcoPassKITBusinessArch.svg:1]: Minor, but noteworthy: .svg seems to have another font than the .jpg

**Review by maxim-bo — COMMENTED** [review]
Overall, its fine, however, two things have been noticed that could be looked at, if necessary

File: docs/page-software-development-view.md
Comment (matbmoser) [reply docs/page-software-development-view.md:1]: The context is added in the PR #24 i will remove this older one from here so we dont have conflicts later

File: docs/resources/adoption-view/adoption-view-EcoPassKITBusinessArch.svg
Comment (matbmoser) [reply docs/resources/adoption-view/adoption-view-EcoPassKITBusinessArch.svg:1]: I will remove the svg

**Pull request merged**